distributed/distributed.yaml

```diff
@@ -74,6 +74,13 @@ distributed:
   deploy:
     lost-worker-timeout: 15s  # Interval after which to hard-close a lost worker job
 
+  adaptive:
```

Comment:
@mrocklin

Reply:
This config bit seems fine to me?

Reply:
Whoops, thought I deleted that, sorry :)

Reply:
For as long as you're handling everything I don't think you get to apologize :)

Thanks Tom.